contrib/ci/distro.sh

```diff
@@ -35,19 +35,48 @@ else
 fi
 declare -r DISTRO_FAMILY
 
-DISTRO_ID=`lsb_release --id | sed -e 's/^[^:]*:\s*\(.*\)$/\L\1\E/'`
+. /etc/os-release
+DISTRO_ID=$ID
 declare -r DISTRO_ID
-DISTRO_RELEASE=`lsb_release --release | sed -e 's/^[^:]*:\s*\(.*\)$/\L\1\E/'`
+DISTRO_RELEASE=$VERSION_ID
 declare -r DISTRO_RELEASE
 
+if [ -f /.dockerenv ]; then
+    IN_CONTAINER=true
+fi
```

Comment:
Remove this and `container_pre_install`:

1. It is not needed anymore since fedora:latest already contains sudo and git is not really needed
2. It is difficult to do it universally, for example there is /run/.containerenv instead of /.dockerenv when run in podman and the command does not have to be run as root therefore you can not install packages

It's not worth it. After we migrate, we'll make these scripts better and less dependent of stuff.